### PR TITLE
updated foreman template - audit fields

### PR DESCRIPTION
**Pull request by lzap**
SSIA Cheers.

### CODE REVIEW

File: namespaces/foreman.yml [hunk 1]
```diff
@@ -5,20 +5,25 @@ namespace:
   description: >
     Contains common fields specific to Foreman / Satellite 6 deployment.
   fields:
-  - name: foreman_logger
+  - name: app
+    type: keyword
+    description: >
+      Appliation which created the log event, one of: foreman, foreman-proxy, dynflow, candlepin or pulp.
```
Comment (richm) [inline namespaces/foreman.yml]: Application

**Comment by lzap** [comment]
I have a question - when I was testing this and searching for foreman.request which is UUID, Kibana/ES actually parsed my query and broke UUID into parts separated by dash characters and then it was searching the tokens as fulltext. It did the job, but how do I tell the stack which fields should be not tokenized?

File: namespaces/foreman.yml (hunk 1 above)
Comment (lzap) [reply namespaces/foreman.yml]: Done thanks, rebased.

**Comment by richm** [comment]
> I have a question - when I was testing this and searching for foreman.request which is UUID, Kibana/ES actually parsed my query and broke UUID into parts separated by dash characters and then it was searching the tokens as fulltext. It did the job, but how do I tell the stack which fields should be not tokenized?

The field is type `keyword` - I thought this means not to tokenize the field - @lukas-vlcek do you know if there is an additional setting needed so that this field is not tokenized?

**Comment by lzap** [comment]
I understand that and it is not being tokenized, however the query is. How do I tell kibana that if I enter "foreman.uuid: XXX-YYYY-ZZZ-ZZZZ" do not tokenize it? I need to retest this, maybe I was just entering "XXX-YYYY-ZZZZ-ZZZZ" without field specification and then tokenization is correct for sure.

However this should be ready for merge, thanks!

**Comment by richm** [comment]
> I understand that and it is not being tokenized, however the query is. How do I tell kibana that if I enter "foreman.uuid: XXX-YYYY-ZZZ-ZZZZ" do not tokenize it? I need to retest this, maybe I was just entering "XXX-YYYY-ZZZZ-ZZZZ" without field specification and then tokenization is correct for sure.
> 
> However this should be ready for merge, thanks!

So it is just Kibana that is doing the tokenization?  If you use curl to do the query (e.g. something like `/*indexpattern*/_search?q=foreman.uuid:XXX-YYYY-ZZZ-ZZZZ\&pretty` does it do an exact match search for the `XXX-YYYY-ZZZ-ZZZZ` value?  If so, then it is probably related to Kibana index patterns - you will have to go into the Kibana index patterns configuration, add this field, and configure it to be a "keyword" instead of a "text" field (or whatever terminology Kibana uses).

**Comment by lzap** [comment]
I believe that Lucene does all tokenization, it must be the same tokenization engine which also returns results otherwise it could return wrong data. Anyway, this is now ready. Thanks!

**Review by richm — APPROVED** [review]
/lgtm

**Pull request merged**